### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/membrane_opus.ex [hunk 1]
```diff
@@ -8,31 +8,31 @@ defmodule Membrane.Opus.Description do
   @typedoc """
   Number of channels transmitted in the stream.
   """
-  @type channels_t :: 1 | 2
+  @type channels_t :: 1 | 2 | nil
```
Comment (bblaszkow06) [inline lib/membrane_opus.ex:11]: So far we didn't allow that (`nil`s in caps). Do you want to change that? `Membrane.Opus` struct with nils would mark unparsed stream? Still it seems not usable as File.Source would not provide such format in caps

**Review by bblaszkow06 — CHANGES_REQUESTED** [review]
* README requires changes
* CI is not enabled

File: lib/membrane_opus.ex (hunk 1 above)
Comment (mat-hek) [reply lib/membrane_opus.ex:11]: File.Source would not, but RTP.Depayloader does. At the same time, it outputs the stream chunked by Opus packets.

**Comment by mat-hek** [comment]
CI will be fixed once the new docker is ready

File: lib/membrane_opus.ex [hunk 2]
```diff
@@ -8,31 +8,31 @@ defmodule Membrane.Opus.Description do
...
 
   @typedoc """
-  Enable/disable in-band _Forward Error Correction_ (FEC).
+  Determines whether the stream is equipped with in-band _Forward Error Correction_ (FEC).
   """
-  @type fec_enabled_t :: boolean
+  @type fec_available_t :: boolean | nil
```
Comment (bblaszkow06) [inline lib/membrane_opus.ex]: Is this actually needed? I haven't found a way to detect that FEC is enabled, so for RTP received streams we'll never know and when encoding it is enough to set it as encoder option

File: lib/membrane_opus.ex (hunk 1 above)
Comment (bblaszkow06) [reply lib/membrane_opus.ex:11]: Maybe the Depayloader should handle that? Getting channels and bitrate and channels is just a matter of interpreting first byte of this packet

Comment (mat-hek) [reply lib/membrane_opus.ex:11]: I believe it should be done by the parser, but the parser isn't always needed

**Comment by mkaput** [comment]
> CI will be fixed once the new docker is ready

@mat-hek  what's missing in the image I made? :)

**Comment by mat-hek** [comment]
@mkaput I suppose it wasn't finished 11 days ago, now its added to CI and nothing seems to be missing :P

**Review by bblaszkow06 — APPROVED** [review]

**Pull request merged**